### PR TITLE
Betterserver - Impl 1

### PR DESCRIPTION
**Pull request by postprandial**
Implementation 1 as done on Sunday, after discussion and experiments on Tuesday
contains a few syntax comments for note-taking purposes, we might delete this later

### CODE REVIEW

File: content/project/impl_1/rcp.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from jsonschema import validate
+import time
+import json
+
+
+class EchoHandler(BaseRequestHandler):
+    def handle(self):
+        print('Got connection from', self.client_address)
+        while True:
+            msg = self.request.recv(8192)
```
Comment (abigailcx) [inline content/project/impl_1/rcp.py:11]: @postprandial Let's add a comment to this line saying: "8192 refers to the number of received bytes to get from the request in one data block"

**Review by abigailcx — CHANGES_REQUESTED** [review]
Just want to add a note about the request.recv() line, otherwise LGTM!

File: content/project/impl_1/rcp.py (hunk 1 above)
Comment (postprandial) [reply content/project/impl_1/rcp.py:11]: Good idea!

Comment (postprandial) [reply content/project/impl_1/rcp.py:11]: Added comment @abigailcx

**Review by abigailcx — APPROVED** [review]
LGTM - thanks!

**Pull request merged**